src/index.html

```diff
@@ -15,18 +16,50 @@
       rel="stylesheet"
       href="style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
Attributes within the <link> tag should be ordered consistently. You should place 'rel' before 'href' for readability and consistency.